contributing.md

```diff
@@ -6,7 +6,13 @@ Start a new [Discussion](https://github.com/tendermint/starport/discussions/new)
 
 Thank you for your contribution!
 
-## Opening pull requests 
+## Providing Feedback
+
+* Before you open an issue, do a web search, and check for [existing open and closed GitHub Issues](https://github.com/tendermint/starport/issues) to see if your question has already been asked and answered. If you find a relevant topic, you can comment on that issue.
```

Comment:
What about navigating users to the [Discussions ](https://github.com/tendermint/starport/discussions) page instead and only use issues for direct bug reports and feature requests?

So, basially, we can convert these three bullets in two, first one saying that "please open a discussion or comment to an existent one for your questions, feedbacks, ideas or use the Discord channel for quick responses."

Second, "please create issues from this link to report a bug or ask for a feature request."

Reply:
@ilgooz [Discussions](https://github.com/tendermint/starport/discussions) pages are great alternative but our large user-base/community is over Discord, it supports group voice channels if needed + Screen sharing available in groups!

Reply:
We'll think about Discord vs alternatives later. Right now Discord has traction, so we're using that. If people will start using discussions (it's open for them now), we might reconsider.